geomet_data_registry/layer/cansips.py

```diff
@@ -150,5 +150,17 @@ def identify(self, filepath):
 
         return True
 
+    def add_time_key(self):
+        """
+        TO BE DONE
+
+        Add time keys when applicable:
+            - model run default time
+            - model run extent
+            - forecast hour extent
+        and for observation:
+            - latest time step
+        """
+
```

Comment:
1. these should not be blank (add `pass` at least)
2. Mark the section as `TODO` per https://www.python.org/dev/peps/pep-0350/

Reply:
added #TODO and pass